.drone.yml

```diff
@@ -104,4 +104,4 @@ steps:
 - name: assembly/site with OpenJDK11
   image: maven:3.6.3-adoptopenjdk-11
   commands:
-  - mvn -e package -Passembly && mvn -e site -Dlinkcheck.skip=true
+  - mvn -e --no-transfer-progress package -Passembly && mvn -e site -Dlinkcheck.skip=true
```

Comment:
Second maven execution also need it as it will download related to site plugins

Reply:
I was not sure how to satisfy line length rule because this line requires split. I was going to try different options later but it looks like I will do it in other PR :)

Reply:
Just do regular line warp with "&&"